ci/recipe/meta.yaml

```diff
@@ -33,6 +33,8 @@ requirements:
 test:
   requires:
     - pytest
+    - tornado
+    - notebook
```

Comment:
Well, wasn't expecting that!

I think this is still a good thing, as it at least now proves the jupyter extension module is importable..., although they don't have any doctests lol.